xdsl/irdl.py

```diff
@@ -763,8 +762,8 @@ def get_variadic_sizes(op: Operation, op_def: OpDef,
 def get_operand_result_or_region(
     op: Operation, op_def: OpDef, arg_def_idx: int, previous_var_args: int,
     construct: VarIRConstruct
-) -> None | SSAValue | FrozenList[SSAValue] | list[OpResult] | Region | list[
-        Region]:
+) -> None | SSAValue | tuple[SSAValue,
+                             ...] | list[OpResult] | Region | list[Region]:
```

Comment:
Must admit, this is once again a moment where I would prefer a different formatter....

Reply:
:)